base/mqueue/mqueue.go

```diff
@@ -35,16 +35,17 @@ type writerImpl struct {
 func ReaderFromEnv(topic string) Reader {
 	kafkaAddress := utils.GetenvOrFail("KAFKA_ADDRESS")
 	kafkaGroup := utils.GetenvOrFail("KAFKA_GROUP")
+	minBytes := utils.GetIntEnvOrDefault("KAFKA_READER_MIN_BYTES", 1)
+	maxBytes := utils.GetIntEnvOrDefault("KAFKA_READER_MAX_BYTES", 1e6)
```

Comment:
Seems fine, But we need to e cautious, since the MAX_BYTES variable will siginficantly affect memory usage

Reply:
There is no reason to change the value in production, but it turned out in perf-testing it's better to have a possibility to change it.